server/common/default_config.py

```diff
@@ -60,6 +60,10 @@
   enable: true
   lfc_cutoff: 0.01
 
+data_locator:
```

Comment:
One improvement, I think, would be to have one place to define the region name.
Right now I believe we need to set the region in the data_locator and the region in the tiledb context.

Reply:
> Looks good to me, but there may be an improvement to consider to reuse the same region name
> for both the data locator and tiledb.

Yep, I thought about this but ran into some uncertainty about how best to do it.  How about I'll land this PR, but file an issue to do the cleanup in the near future?